config/templates/filters.conf.njk

```diff
@@ -8,6 +8,7 @@
     Rename level log.level
     Rename event_sequence event.sequence
     Rename message event.original
+    Rename message_parsed message
```

Comment:
If you can, move this and the line above to the tomcat filter. config/templates/tomcat/filter/filters.conf.njk

Global ones are iffy.

Reply:
Good point, I will make that change.

Reply:
@mbystedt I moved the lines to the application section of the filter file. It worked locally, but wanted to let you know in case you see a better place.